revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+# 10 Vision of Success
+
+This proposal aims not to simply funnel granted funds to increase the platform’s volume and liquidity. Instead, it seeks to spread them more thoroughly and thoughtfully to primarily cater to user expectations and demands to enjoy an excellent DeFi experience.
+
+While the number of users is easy to count, it is much more challenging to predict the user base growth rate. User adoption depends not only on the success and functionalities of the platform but also on factors beyond the team’s control, such as network stability, market sentiment, competition, media presence, and word of mouth.
+
+At the beginning of 2023, Beamswap had an average of 2,661 unique users per month and added 953 new users per month. These figures will be used as a baseline for determining the Incentive Program’s success along with TVL, volume, and transaction count.
+
+As Beamswap progresses with platform development, continuously steady QOQ user growth is anticipated, especially with incentives powering the user onboarding initiatives and campaigns.
+
+![|602x339](upload://91jHDEqQmYsNh2v469wzV5IF4Ac.jpeg)
+
+Ultimately, the long-term goal for Beamswap is to become a fee-powered DEX, reliant only on ongoing user activity bringing in sufficient volume and liquidity.
+
+# 11 Rationale
+
+Beamswap is about to introduce two new products - concentrated liquidity AMM and perpetual DEX. Compared to a standard AMM, these will enhance transactional efficiency while lowering operating costs. However, their operation requires a higher percentage of liquidity, as seen with concentrated liquidity and perpetual solutions on other networks.
+
+By bringing new users to the Beamswap platform and familiarizing them with the network’s benefits, the Moonbeam ecosystem would get a significant boost in daily volume and liquidity, further solidifying its relevance as the go-to network for builders, users, and service providers.
+
+As more XC-20 assets are supported, Beamswap continues reinforcing the implementation of Moonbeam’s connected contract capabilities. Better asset choices and cross-chain connectivity would provide users with greater DeFi options, boosting market depth. Simultaneously, the optimized rates and mitigated slippage would improve user experience and safeguard their portfolio.
+
+In the case of grant application approval, the Beamswap team plans to roll out advanced features and platform upgrades to pursue the expansion and loyalty of the DeFi community. On the one hand, these would reward users for their activities, either trading crypto assets or promoting the platform. On the other hand, they would optimize the platform offering and expand its reach with both Beamswap-launched cutting-edge solutions like concentrated liquidity AMM and perpetual DEX, as well as those supplied by partners, such as the cross-chain liquidity routing module by Axelar and Squid.
+
+![|602x339](upload://3toEfKHojF1afpZQrjK0LvBFEQG.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.